log/logger_types.go

```diff
@@ -75,8 +89,8 @@ type loggerFileConfig struct {
 
 // Logger each instance of logger settings
 type Logger struct {
-	ShowLogSystemName                                bool
-	Timestamp                                        string
+	ShowLogSystemName, BypassJobChannelFilledWarning bool
```

Comment:
I'd classify `BypassJobChannelFilledWarning` as a different kind of `bool` than `ShowLogSystemName` so please put it on a different line.